### ISSUE TITLE
HOURS in filename shows ridiculous high value

### ISSUE THREAD

**Issue by GoogleCodeExporter** [achim1/muonic#16]
```
After a long measurement, the calculated HOURS stamp is sometimes way to large 
e.g of the order of 1e5, allthough the measurement was not active for that long.

```

Original issue reported on code.google.com by `achim.st...@gmail.com` on 14 Jun 2013 at 11:29

**Comment by achim1** [comment]
Is this problem still existing in Version 2.0.6 and beyond?

**Comment by hpbretz** [comment]
This is one of 3 issues marked as bug, so it should probably be addressed before 2.1. Has anyone still experienced this issue? Could someone maybe look at stome of the long term measurements recently done?

**Comment by aschulzdesy** [comment]
Testing with version 2.0.12: still there: 
you see from the L file that the measurement was started on july 17th and ran until july 21, so around 100 hours. 

![bildschirmfoto vom 2015-07-21 10_41_54](https://cloud.githubusercontent.com/assets/4849214/8796490/6fec6376-2f95-11e5-8d38-fba8d7e14f27.png)
